test/Microsoft.Azure.SignalR.E2ETests/Management/ServiceHubContextE2EFacts.cs

```diff
@@ -130,15 +130,15 @@ internal async Task TestAddUserToGroupWithTtl(ServiceTransportType serviceTransp
                     clientEndpoint,
                     clientAccessTokens,
                     () => SendToGroupCore(serviceHubContext, userGroupDict, SendAsync, (c, d) => AddUserToGroupWithTtlAsync(c, d, TimeSpan.FromSeconds(10)), Empty),
-                    _userNames.Length / _groupNames.Length + _userNames.Length % _groupNames.Length,
+                    (_userNames.Length / _groupNames.Length + _userNames.Length % _groupNames.Length) * 2,
```

Comment:
looks like `SendAsync()` only send to the 1st group, each group has 2 connections, so 2 should be expected?

Reply:
send twice: [first](https://github.com/Azure/azure-signalr/blob/54570018cc8b19f31e5f5afdbe01a696c458fd82/test/Microsoft.Azure.SignalR.E2ETests/Management/ServiceHubContextE2EFacts.cs#L351) [second](https://github.com/Azure/azure-signalr/blob/54570018cc8b19f31e5f5afdbe01a696c458fd82/test/Microsoft.Azure.SignalR.E2ETests/Management/ServiceHubContextE2EFacts.cs#L355)